fragments/teardown/ceph_ansible_purge.yaml

```diff
@@ -0,0 +1,9 @@
+tests:
```

Comment:
IMO, we should have `fragments/teardown/ceph-ansible/purge.yaml` like the way use in install